db/migrate/20250203180045_add_can_share_crafter_details_to_projects.rb

```diff
@@ -1,5 +1,5 @@
 class AddCanShareCrafterDetailsToProjects < ActiveRecord::Migration[7.0]
   def change
-    add_column :projects, :can_share_crafter_details, :boolean, default: false
+    add_column :projects, :can_share_crafter_details, :boolean, default: true
```

Comment:
I think you will be better off with a new migration that uses `change_column_default`. That way it will deploy automatically